sdmetrics/single_column/statistical/kscomplement.py

```diff
@@ -57,6 +59,13 @@ def compute(real_data, synthetic_data):
             real_data = pd.to_numeric(real_data)
             synthetic_data = pd.to_numeric(synthetic_data)
 
+        try:
```

Comment:
The `try/catch` ensures invalid data raises the same error as before, ie it crashes during `ks_2samp`, not here. Without it various tests would need to be updated to use the error raised by `round` instead, which happens to be different for older versions of numpy and recent ones.

Reply:
what are the situations where we can't round?

Reply:
should we silently pass in those scenarios?

Reply:
We could add more input validation, but that seems out of scope of this task, since it affects the behavior of the function besides what's described in the issue

Reply:
Yes, it's just a little tricky because different errors show up for different numpy versions, so the tests need to take that into consideration. Let me know if that's what you prefer and I'll implement it.